biofefi/components/experiments.py

```diff
@@ -21,3 +21,22 @@ def experiment_selector(options: list) -> Path:
         placeholder="Experiment name",
         key=ViewExperimentKeys.ExperimentName,
     )
+
+
+def model_selector(options: list) -> Path:
+    """Select
```

Comment:
Finish this docstring please.

Reply:
Hey! I am not sure of what you mean with this...

Reply:
It just says "Select" at the top. "Select" what?